dev/setup-rancher-clusters

```diff
@@ -19,8 +19,13 @@ rancherpassword="${RANCHER_PASSWORD-rancherpassword}"
 
 public_hostname="${public_hostname:-}"
 if [ -z "${public_hostname}" ]; then
-  until kubectl get service -n kube-system traefik -o jsonpath='{.status.loadBalancer.ingress[0].ip}'; do sleep 3; done
-  ip=$(kubectl get service -n kube-system traefik -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  success=-1
+  until [ "$success" -eq 0 ]; do
+    echo "Waiting for Traefik LoadBalancer IP..."
+    sleep 5
```

Comment:
nit: is there a specific reason for a `sleep 5` where the previous `until...do` construct used a `sleep 3`?